library/Icinga/Protocol/Ldap/LdapUtils.php

```diff
@@ -62,11 +64,42 @@ public static function implodeDN($parts)
     }
 
     /**
+     * Test if supplied value looks like a DN
+     *
+     * Note: This is a rather complex regex, open to better solutions!
+     *
+     * See test for examples.
+     *
+     * @param  mixed $value
+     *
+     * @return bool
+     */
+    public static function isDN($value)
```

Comment:
Please rename the function to `isDn`. I know that the other functions have a capital N but this is just lame. We call them with the lowercase n anyway. So it would make sense to rename the other functions as well. No bc here, PHP does not care about case sensitiveness when calling methods.
Please remove the note and reference to tests from the doc of the function.
Instead of implementing a custom logic for validating DNs which is prone to errors (no offence, just look at this [beast](https://github.com/zendframework/zend-ldap/blob/master/src/Dn.php#L588)) I would only check whether `ldap_dn2ufn` returns not false.